lib/chat_api/event_subscriptions.ex

```diff
@@ -8,6 +8,9 @@ defmodule ChatApi.EventSubscriptions do
 
   alias ChatApi.EventSubscriptions.EventSubscription
 
+  @type tesla_reponse() :: {:ok, Tesla.Env.t()} | {:error, any()}
```

Comment:
just curious, does `Tesla` not have this type built in?

Reply:
will look at it